Source/lua/modules/audio.cpp

```diff
@@ -0,0 +1,25 @@
+#include "lua/modules/render.hpp"
+
+#include <sol/sol.hpp>
+
+#include "effects.h"
+
+namespace devilution {
+
+namespace {
+
+bool IsValidSfx(int16_t psfx)
+{
+	return psfx >= 0 && psfx <= static_cast<int16_t>(SfxID::LAST);
+}
+
+} // namespace
+
+sol::table LuaAudioModule(sol::state_view &lua)
+{
+	return lua.create_table_with(
+	    "playSfx", [](int psfx) { if (IsValidSfx(psfx)) PlaySFX(static_cast<SfxID>(psfx)); },
+	    "playSfxLoc", [](int psfx, int x, int y) { if (IsValidSfx(psfx)) PlaySfxLoc(static_cast<SfxID>(psfx), { x, y }); });

```

Comment:
Should we allow single-line functions and single-line if statements throughout the codebase?
I think it'd suit our current style pretty well.
Especially in places like https://github.com/diasurgical/devilutionX/blob/7b8dad456eab73765f9e36e6ed2b257573680641/Source/panels/console.cpp#L282-L308

Reply:
I think a big part of it is me not being familiar with arrow functions, they already look incredibly mess to me.

I don't see any if statements in the code you quoted?

Reply:
In this code there are none but we have a ton of code like this:
```c
if (!ok)
  return;
```

Reply:
I'm not strongly against 1 lining them.